### ISSUE TITLE
Create a custom `_meilisearch_id` field instead of overwriting the existing `id` field.

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/strapi-plugin-meilisearch#609]
See #608 

Currently, before an entry is indexed into Meilisearch, the plugin pre-prend its `id` with the collection name.
For example:

```
Restaurant {
id: 1,
name: "Pizza Freak" 
}
```

In Strapi becomes:

```
Restaurant {
id: "restaurant-1",
name: "Pizza Freak" 
}
```

In Meilisearch.

The user is not aware of this change and it might lead to confusion as it is not well documented.

To solve this issue, instead of overwriting the `id` field, the plugin should create a new field named `_meilisearch_id` containing the transformed value of the id:

```
Restaurant {
id: 1,
_meilisearch_id: "restaurant-1",
name: "Pizza Freak" 
}
```

## TODO
- Change `id` to `_meilisearch_id` here
https://github.com/meilisearch/strapi-plugin-meilisearch/blob/354db6c888648412c339fb592ba836f726daea59/server/services/meilisearch/adapter.js#L37-L45
- In every occurrence of `client.index(indexUid).addDocuments(documents)` the primary key should be specifically provided. [See doc](https://docs.meilisearch.com/reference/api/documents.html#add-or-replace-documents)
```js
client.index(indexUid).addDocuments(documents, { primaryKey: "_meilisearch_id" })
```
- add tests

**Comment by bidoubiwa** [comment]
Of course @Pablo-Aldana ! Thanks a lot

**Comment by bidoubiwa** [comment]
fixed by #612
